tools/tplg_parser/include/tplg_parser/topology.h

```diff
@@ -206,5 +206,7 @@ bool tplg_is_valid_priv_size(size_t size_read, size_t priv_size,
 int tplg_create_object(struct tplg_context *ctx,
 		       const struct sof_topology_module_desc *desc, int num_desc,
 		       const char *name, void *object, size_t max_object_size);
-
+int sof_parse_token_sets(void *object, const struct sof_topology_token *tokens,
+			 int count, struct snd_soc_tplg_vendor_array *array,
+			 int priv_size, int num_sets, int object_size);
```

Comment:
I must admit I don't even know what the tplg_parser is or does, so stopping the review here. Way beyond my understanding.

Reply:
@plbossart the tplg_parser is a user-space library for parsing the topologies and it is used by the testbench today. And it will also be utilized for the alsa plugin that we are current working on.